app/helpers/application_helper.rb

```diff
@@ -3,4 +3,13 @@ def full_title page_title
     base_title = t "home.title"
     page_title.blank? ? base_title : page_title + " | " + base_title
   end
+
+  def format_date date
+    date.to_time.strftime t "datetime.formats.default"
```

Comment:
I18n.l(date.to_time) Thử đổi thành thế này xem có chạy ko ông

Reply:
Nó chạy bình thường, tuy nhiên tôi không muốn format theo cái đó ô ạ.
Tôi muốn custom en: m/d/Y - H:M, vi: d/m/Y - H:M ô ạ

Reply:
cái này nó có thay đổi theo ngôn ngữ k e?
lúc nào cũng là 1 giá trị thì phải move cái t "datetime.formats.default" vào setting chứ e
còn muốn hiển thị theo locale thì dùng 
https://stackoverflow.com/questions/36881411/localizing-datetimes-in-rails-i18n

Reply:
có format theo local chị ạ